app/database_etl/location_utils/location_util_functions.py

```diff
@@ -75,6 +75,10 @@ def get_country_code(country_name, iso3=True):
     return code
 
 
+def get_alternative_names(country_code):
+    return COUNTRY_ALTERNATIVE_NAMES[country_code] if country_code in COUNTRY_ALTERNATIVE_NAMES else None
```

Comment:
I don't think we need to make a global constant. Just read the file in this function instead of outside

Reply:
In this case, we need to read the file into memory every time the function is called. It's better design to read it in once, outside the function